src/main/resources/aws/2-nodes-dc.yaml

```diff
@@ -183,6 +184,17 @@ Resources:
           FromPort: 3306
           ToPort: 3306
           CidrIp: 0.0.0.0/0
+  PostgresSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
```

Comment:
The mini-disadvantage is that it will allocate two Security Groups even tho only one is used. It's an AWS resource, so it'll cost a little and might increase CloudFormation provisioning time. OTOH SecurityGroups are relatively cheap, so I guess we're fine here. Just FYI.